<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cframe</class>
 <widget class="QMainWindow" name="cframe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1543</width>
    <height>858</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cframe</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1541</width>
      <height>811</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>110</y>
        <width>281</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="Imagenes.qrc">:/Imagenes/Canvas-Logo-transparent-Large.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditUsuario">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>280</y>
        <width>361</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sans Serif Collection</family>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditPassword">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>360</y>
        <width>361</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sans Serif Collection</family>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>260</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Correo:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>340</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Contraseña:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_Login">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>470</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ingresar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_registrar">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>390</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>No tienes una cuenta?</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnVerPasswordLogin">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>360</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Imagenes.qrc">
        <normaloff>:/Imagenes/ocultarcontra.png</normaloff>:/Imagenes/ocultarcontra.png</iconset>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>670</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Volver</string>
      </property>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget_2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>100</y>
        <width>1121</width>
        <height>631</height>
       </rect>
      </property>
      <widget class="QWidget" name="page_5">
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>120</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI Semibold</family>
          <pointsize>10</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Apellido:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditConfirmarPassword">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>360</y>
          <width>271</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_23">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>340</y>
          <width>161</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI Semibold</family>
          <pointsize>10</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Confirmar contraseña:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_22">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>340</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI Semibold</family>
          <pointsize>10</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Contraseña:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditNombre">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>140</y>
          <width>271</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxTipoUsuario">
        <property name="geometry">
         <rect>
          <x>840</x>
          <y>250</y>
          <width>82</width>
          <height>28</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Registro</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Maestro</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Alumno</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="btn_CrearUsuario">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>490</y>
          <width>91</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Crear Usuario</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditApellido">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>140</y>
          <width>271</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditNuevoUsuario">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>250</y>
          <width>351</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_21">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>230</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI Semibold</family>
          <pointsize>10</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Usuario:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>119</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI Semibold</family>
          <pointsize>10</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Nombre:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditNuevoPassword">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>360</y>
          <width>271</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_6"/>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>340</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>81</width>
        <height>761</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>:/Imagenes/Barra canvas.PNG</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnsalir">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>730</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::ContextMenuPolicy::PreventContextMenu</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LayoutDirection::LeftToRight</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>1450</x>
        <y>180</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1543</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
